packages/grid/components.ts

```diff
@@ -0,0 +1,14 @@
+import { StarryUIComponentDefinition } from '@starryui/traits'
+import { grid } from '.'
+import { withTextContent } from '@starryui/traits'
+import { applyTheme } from '@starryui/theme'
+
+export const buttonDefinition: StarryUIComponentDefinition = {
```

Comment:
Looks like leftover reference to `button` here

Reply:
To get the grid into the docs you need to add it to https://github.com/starryui/starryui/blob/main/packages/starryui-docs/pages/components/component-list.ts

Reply:
Done

Reply:
✅ Perfect